Spec/RealtimeClientConnection.swift

```diff
@@ -1169,6 +1169,32 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN13
+            context("Ping") {
+
+                // RTN13b
+                pending("should immediately indicate an error if in the CLOSED or FAILED state") {
+                    let client = AblyTests.newRealtime(AblyTests.commonAppSetup())
+                    defer { client.close() }
+                    waitUntil(timeout: testTimeout) { done in
+                        client.connection.on(.Connected) { _ in
+                            client.onError(AblyTests.newErrorProtocolMessage())
+                            done()
+                        }
+                    }
+                    waitUntil(timeout: testTimeout) { done in
+                        client.ping() { error in
+                            expect(error).toNot(beNil())
+                            let transport = client.transport as! TestProxyTransport
+                            expect(transport.protocolMessagesSent.filter{ $0.action == .Heartbeat }).to(haveCount(1))
```

Comment:
Why is this one when the spec says it should immediately indicate an error, if the connection is `CLOSED` why are we sending a message?